sql-cli/include/base/config/dev/configuration.yml

```diff
@@ -9,3 +9,7 @@ connections:
   - conn_id: sqlite_conn
     conn_type: sqlite
     host: retail.db
+
+# Apache Airflow optional settings
+airflow:
+  home:
```

Comment:
@pankajkoti I'd love your thoughts on this change, since you worked a lot on this part of the code as well